### PR TITLE
feat: add object-curly-spacing rule

### PR DESCRIPTION
**Pull request by trs**
Requires spaces around objects:

```
{"key": "value"} --> { "key": "value" }
```

Makes one-line objects clearer and easier to read/parse.

This is fixed automatically (if your eslint is set to fix automatically)

### CODE REVIEW

File: eslintrc [hunk 1]
```diff
@@ -24,8 +31,7 @@
       "error",
       2,
       {
-        "SwitchCase": 1,
-        "MemberExpression": "off"
```
Comment (trs) [inline eslintrc]: This has been fixed in eslint to be back to off by default (https://github.com/eslint/eslint/commit/c7488ac6fe47d31f57fbdf72f3ef14fc4979486f)

**Comment by forstermatth** [comment]
@trs @Jl14Salvador I've never personally been a fan of the extra spaces, but if we have a majority of readers who think it makes it easier to read then we can go ahead and merge this.

**Comment by trs** [comment]
@forstermatth That's fair, just looks wise or readability wise?
I'm okay with either way, just figured this might be easier.

**Comment by Jl14Salvador** [comment]
@trs  @forstermatth I would say i like no spaces more, but it doesn't matter for me, either or works.

**Comment by trs** [comment]
image

**Pull request closed without merging**